wiki/Accessibility-A11y-Guide.md

```diff
@@ -85,6 +86,9 @@ TalkBack is the Google **screen reader** included on Android devices. TalkBack g
 * [Getting started with Talkback](https://support.google.com/accessibility/android/answer/6283677)
 * [Display speech output as Text: Talkback](https://developer.android.com/guide/topics/ui/accessibility/testing#optional_talkback_developer_settings)
 
+### Developers Videos
+* [How to use Accessibility Scanner? - Tutorial](https://youtu.be/LF5AgGI5H3A)
```

Comment:
Based on my comment above, this could go under the "**Using a11y scanner in android**" section.

Reply:
Done.